lib/uploader.ts

```diff
@@ -201,7 +201,18 @@ export class Uploader {
 
 					// Init request queue
 					const request = () => {
-						return uploadData(`${tempUrl}/${chunk+1}`, blob, upload.signal, () => this.updateStats(), destinationFile)
+						return uploadData(
+							`${tempUrl}/${chunk+1}`,
+							blob,
+							upload.signal,
+							() => this.updateStats(),
+							destinationFile,
+							{
+								'X-OC-Mtime': file.lastModified,
+								'OC-Chunked': true,
```

Comment:
```suggestion
```

At least this line is not necessary as this is the old chunking mechanism from before https://github.com/owncloud/core/pull/20118